From 15cc525406cf311399d182ffcc18711651b7c8b2 Mon Sep 17 00:00:00 2001
From: Richard Oliver <richard.oliver@raspberrypi.com>
Date: Fri, 12 Jul 2024 13:32:30 +0100
Subject: [PATCH 1247/1350] dts: bcm{283x,2712}: /clocks as "simple-bus"

Make /clocks node compatible with "simple-bus" to ensure that all children
(not just those that are "fixed-clock" compatible) are automatically
probed.

Signed-off-by: Richard Oliver <richard.oliver@raspberrypi.com>
---
 arch/arm/boot/dts/broadcom/bcm283x.dtsi   | 4 ++++
 arch/arm64/boot/dts/broadcom/bcm2712.dtsi | 4 ++++
 2 files changed, 8 insertions(+)

--- a/arch/arm/boot/dts/broadcom/bcm283x.dtsi
+++ b/arch/arm/boot/dts/broadcom/bcm283x.dtsi
@@ -478,6 +478,10 @@
 	};
 
 	clocks {
+		compatible = "simple-bus";
+		#address-cells = <1>;
+		#size-cells = <0>;
+
 		/* The oscillator is the root of the clock tree. */
 		clk_osc: clk-osc {
 			compatible = "fixed-clock";
--- a/arch/arm64/boot/dts/broadcom/bcm2712.dtsi
+++ b/arch/arm64/boot/dts/broadcom/bcm2712.dtsi
@@ -1258,6 +1258,10 @@
 	};
 
 	clocks {
+		compatible = "simple-bus";
+		#address-cells = <1>;
+		#size-cells = <0>;
+
 		/* The oscillator is the root of the clock tree. */
 		clk_osc: clk-osc {
 			compatible = "fixed-clock";
